articles/understanding-the-basics-of-responsive-web-design/index.md

```diff
@@ -0,0 +1,118 @@
+Many global businesses are building websites to provide a platform for marketing and promoting their products. Building an attractive website only may not be enough since customers require a site that promotes usability and responsiveness. Responsive web design ensures that a website responds well to various devices used by customers. A responsive website is important because it enhances great customer experience, drives mobile traffic, and promotes high conversion rates. 
+
+This article takes you through the basics of responsive web design, including its building blocks and why it is important for your business. It also provides a brief guide on how to get started with this technique. Real-life examples of responsive websites have been provided to improve our understanding of responsive web design.  
+
+### What is Responsive Web Design (RWD)?
+In the recent past, we have experienced the growth of devices that allow us to gain access to various websites. These include laptops, desktop computers, tablets, smart TVs, and smartphone devices. When building a website, developers should consider designs that correspond to the screen dimensions and usability of these devices. This is where responsive web design (RWD) comes in. 
+
+A responsive web design is an approach or practice in web development that enables a web page to be responsive to the elements of various devices (eg. layout, screen resolution, and screen width). Through RWD, a website can work on any device. 
+
+Let’s assume that you are using your mobile phone to access a certain website. If the website does not show up precisely or there is limited interaction like in the case of a desktop, then it is likely that the website was not built with RWD. 
+
+Responsive web design utilizes various elements that adjust the layout of a website to suit the orientation and screen size of a specific device (eg. fluid layouts, CSS codes, HTML5, and media queries). This enables users to interact with the website seamlessly. 
+
+### The Building Blocks of RWD
+The following are the main foundations of responsive web design.
+
+#### HTML and CSS 
+A combination of Hypertext Markup Language (HTML) and Cascading Styles Sheet (CSS) helps in controlling and adjusting the layout of a webpage. HTML controls the content, structure, and elements of a webpage. CSS technology edits the layout and design of elements that were created using HTML. 
+
+#### Fluid Grids
+This is an important principle of responsive design that uses relative measurement units (eg. percentages), rather than fixed units (like pixels). It enables the webpage to be resized to correspond with any screen size. With this component, a web page can adapt to the screen size or width of any device. 

```

Comment:
You should mention some of these relative units like `vw` and `vh`. And also define their unit:

Example: 1 viewport width = 1/100th the width of the device.

Reply:
Done.